psalm-baseline.xml

```diff
@@ -85,9 +85,11 @@
         </UndefinedGlobalVariable>
     </file>
     <file src="src/Resources/skeleton/AdminController.tpl.php">
```

Comment:
wouldn't it make sense to exclude `/src/Resources/skeleton/*` ?

Reply:
Yes, in fact, IMO these files should be renamed as `filename.php.tpl` since they are not really `php` files as they are, they are `tpl`, but I'm not sure if that would work or if is something that has to be changed in Symfony.

Reply:
Can you open an issue on maker-bundle about this?

Reply:
Nevermind, they are actually php files 😅, I've excluded the folder.